profile/README.md

```diff
@@ -0,0 +1,33 @@
+# Profiling
+
+This module provides utility code to run benchmarking on features of pyam.
+
+## Dependencies and usage
+
+The profiling module uses [pytest-monitor](https://pytest-monitor.readthedocs.io)
+package.
+
+### Installation
+
+In addition to an installation of **pyam** with all optional dependencies,
```

Comment:
Similar to the previous reply: I'd keep all changes related to profiling contained in this folder for now, to avoid that we switch to another profiling solution but have stray/outdated references in other parts of the package.